flocker/control/httpapi.py

```diff
@@ -568,14 +606,24 @@ def container_configuration_response(application, node):
     """
     result = {
         "host": node, "name": application.name,
-        "image": application.image.full_name
+        "image": application.image.full_name,
+        "restart_policy": {
```

Comment:
I mulled over what to do in respect of responses for a while and eventually decided the best thing is to always return the full information about a container in a response, so I'll update the other response examples in the docs. In a response, I think it's ok for example to leave out a `ports` object if there are no exposed ports, but there is always a restart policy whether you explicitly gave it in the request or not, so yeah I think restart policy should always be included in the JSON response.